pkg/skaffold/schema/validation/validation.go

```diff
@@ -479,3 +480,27 @@ func validateSingleKubeContext(configs []*latest.SkaffoldConfig) []error {
 	}
 	return nil
 }
+
+// validateCustomTest
+// - makes sure that command is not empty
+// - makes sure that dependencies.ignore is only used in conjunction with dependencies.paths
+func validateCustomTest(tcs []*latest.TestCase) (errs []error) {
```

Comment:
Can we add tests for this function in `validation_test.go`

Reply:
Added tests.